From 056bb31b4529a78e568a1a60fda987ab84a8272f Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Sun, 07 Apr 2019 13:03:50 +0300
Subject: [PATCH] [SW-0] goya: pll: output1 to ref clock

We are going to raise the CPU PLL clock and thus all outputs will
be affected.
The CPU PLL output1 is the input clock for the timer counter.
In order to keep the delay timer at the same frequency, switch the
CPU PLL output1 to reference clock.

Change-Id: I30b2588acc4aa8de330b3b466bf8e58d2895a44d
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/goya/pll.c b/goya/pll.c
index a1ff3c2..df613e0 100644
--- a/goya/pll.c
+++ b/goya/pll.c
@@ -58,7 +58,7 @@
 
 static struct hl_pll_cfg goya_pll_cfg[] = {
 	{.name = "CPU", .base = (void *)mmCPU_PLL_BASE,
-	 .nr = 0, .nf = 47, .od = 1, .nb = 23, .div_factor = {0, 23, 1, -1} },
+	 .nr = 0, .nf = 47, .od = 1, .nb = 23, .div_factor = {0, -1, 1, -1} },
 	{.name = "IC", .base = (void *)mmIC_PLL_BASE,
 	 .nr = 0, .nf = 63, .od = 15, .nb = 31, .div_factor = {0, 1, 1, 2} },
 	{.name = "MME", .base = (void *)mmPSOC_MME_PLL_BASE,
